src/core/Microsoft.PowerToys.Settings.UI.Lib/KeyboardManagerSettings.cs

```diff
@@ -18,21 +14,15 @@ public class KeyboardManagerSettings : BasePTModuleSettings
         public KeyboardManagerSettings()
         {
             Properties = new KeyboardManagerProperties();
-            version = "1";
-            name = "_unset_";
+            Version = "1";
+            Name = "_unset_";

```

Comment:
Ist safe to hard code the name to "Keyboard Manager" or should this be in the strings resource for language translation?

Reply:
That is an @arjunbalgovind question